config.js

```diff
@@ -1118,6 +1118,9 @@ config.DEFAULT_REGION = 'us-east-1';
 
 config.VACCUM_ANALYZER_INTERVAL = 86400000;
 
+config.METRICS_AUTH_ENABLED = true;
+config.VERSION_AUTH_ENABLED = true;
```

Comment:
i am not sure about the default being true. will that break anything?

Reply:
What did you test for? did you check for noobaa-cli health, and upgrades, etc?

Reply:
@guymguym In this PR we are skipping token authentication for NSFS NC. Metrics and version authentication only for the containerized Noobaa.